Source/items.cpp

```diff
@@ -3769,28 +3762,26 @@ void GetItemStr(ItemStruct &item)
 	}
 }
 
-void CheckIdentify(int pnum, int cii)
+void CheckIdentify(PlayerStruct &player, int cii)
 {
 	ItemStruct *pi;
 
 	if (cii >= NUM_INVLOC)
-		pi = &Players[pnum].InvList[cii - NUM_INVLOC];
+		pi = &player.InvList[cii - NUM_INVLOC];
 	else
-		pi = &Players[pnum].InvBody[cii];
+		pi = &player.InvBody[cii];

```

Comment:
Nit: Other functions with this code use braces, inconsistent with the style guide for single line cases.

However given you're touching the lines anyway extracting this to a function returning an `ItemStruct&` would be great. Then we know all four functions are using the same logic and the initialisation can also be made simpler.

Reply:
Saw it, but decided i needed to move on and leave something for someone else :)